<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainForm</class>
 <widget class="QMainWindow" name="CMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>V10Win</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Меню/Прервать</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>151</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Настройки COM порта</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action</sender>
   <signal>activated()</signal>
   <receiver>CMainForm</receiver>
   <slot>handleAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CMainForm</receiver>
   <slot>handlePushButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CMainForm</receiver>
   <slot>handlePushButton_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleAction()</slot>
  <slot>handlePushButton()</slot>
  <slot>handlePushButton_2()</slot>
 </slots>
</ui>
